error: ‘ARMA3’ has not been declared

## Error:
While building the dayz launcher, I get the following output from `make` (this isn't a clean build, but the error message is the same in either case):
```
[ 76%] Built target curlpp
[ 76%] Built target common
[ 76%] Built target steam-integration
[ 76%] Built target arma3-unix-launcher-library
[ 76%] Built target dayz-linux-launcher-library
[ 76%] Built target pugixml
[ 76%] Automatic MOC and UIC for target dayz-linux-launcher
[ 76%] Built target dayz-linux-launcher_autogen
Consolidate compiler generated dependencies of target dayz-linux-launcher
[ 76%] Building CXX object src/dayz-linux-launcher/CMakeFiles/dayz-linux-launcher.dir/main.cpp.o
/home/ropi/Downloads/arma3-unix-launcher/src/dayz-linux-launcher/main.cpp: In function ‘int main(int, char**)’:
/home/ropi/Downloads/arma3-unix-launcher/src/dayz-linux-launcher/main.cpp:151:73: error: ‘ARMA3’ has not been declared
  151 |                             workshop_path = steam.GetWorkshopPath(path, ARMA3::Definitions::app_id);
      |                                                                         ^~~~~
make[2]: *** [src/dayz-linux-launcher/CMakeFiles/dayz-linux-launcher.dir/build.make:166: src/dayz-linux-launcher/CMakeFiles/dayz-linux-launcher.dir/main.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:463: src/dayz-linux-launcher/CMakeFiles/dayz-linux-launcher.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

## Way to reproduce:
Follow the instructions on _Build process for DayZ_

## Environment:
OS: Pop!_OS 22.04
CMake: cmake version 3.22.1
GCC: gcc version 11.3.0 (Ubuntu 11.3.0-1ubuntu1~22.04)
QT: I don't have `qt5-default` installed, but as people [here](https://askubuntu.com/questions/1335184/qt5-default-not-in-ubuntu-21-04) suggested, I installed these packages:
`apt-get install qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools`

--- a/.github/workflows/linux_packages.yml
+++ b/.github/workflows/linux_packages.yml
@@ -61,3 +61,29 @@ jobs:
       with:
         file: "*.AppImage"
         draft: true
+  dayz-appimage-gcc-8:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set permissions
+      run: chmod 777 ${{ github.workspace }}
+    - name: Run the build process with Docker
+      uses: addnab/docker-run-action@v3
+      with:
+        image: muttleyxd/a3ul_appimage_build:latest
+        options: -v ${{ github.workspace }}:/github/workspace
+        run: |
+             set -eux
+             cd /github/workspace
+             tools/ci/packaging/appimage_dayz/build.sh /github/workspace /tmp/build /tmp/build
+             cp /tmp/build/*.AppImage /github/workspace
+    - name: draft release
+      if: github.ref == 'refs/heads/master'
+      uses: xresloader/upload-to-github-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.actionrelease }}
+      with:
+        file: "*.AppImage"
+        draft: true

--- a/.github/workflows/linux_test_builds.yml
+++ b/.github/workflows/linux_test_builds.yml
@@ -67,3 +67,18 @@ jobs:
              cmake /github/workspace -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_CXX_FLAGS="-Werror" -DRUN_TESTS=ON -DDEVELOPER_MODE=ON -DRUN_TESTS_FUZZING=ON -DRUN_TESTS_UNIT=OFF
              make -j$(nproc)
              ctest --output-on-failure || ls | grep crash | xargs -n1 cat
+  dayz-archlinux-gcc:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run the build process with Docker
+      uses: addnab/docker-run-action@v3
+      with:
+        image: muttleyxd/a3ul_archlinux_build:latest
+        options: -v ${{ github.workspace }}:/github/workspace
+        run: |
+             set -euxo pipefail
+             mkdir /tmp/build && cd /tmp/build
+             cmake /github/workspace -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS="-Werror" -DRUN_TESTS=ON -DDEVELOPER_MODE=ON -DBUILD_DAYZ_LAUNCHER=ON
+             make -j$(nproc)
+             ctest -V

--- a/src/dayz-linux-launcher/main.cpp
+++ b/src/dayz-linux-launcher/main.cpp
@@ -148,7 +148,7 @@ int main(int argc, char *argv[])
                         arma_path = steam.GetGamePathFromInstallPath(path, DayZ::Definitions::app_id);
                         try
                         {
-                            workshop_path = steam.GetWorkshopPath(path, ARMA3::Definitions::app_id);
+                            workshop_path = steam.GetWorkshopPath(path, DayZ::Definitions::app_id);
                         }
                         catch (std::exception const &e) // todo: write macro for this? ex. TRY()
                         {

